Mathlib/Algebra/Ring/Invertible.lean

```diff
@@ -45,6 +45,35 @@ theorem invOf_sub_invOf [Ring R] (a b : R) [Invertible a] [Invertible b] :
     ⅟a - ⅟b = ⅟a * (b - a) * ⅟b := by
   rw [mul_sub, invOf_mul_self, sub_mul, one_mul, mul_assoc, mul_invOf_self, mul_one]
 
+theorem eq_of_invOf_add_eq_invOf_add_invOf [Ring R] {a b : R} [Invertible a] [Invertible b]
+    [Invertible (a + b)] (h : ⅟(a + b) = ⅟a + ⅟b) :
+    a * ⅟b * a = b * ⅟a * b := by
+  have h_neg_identity : -1 = ⅟a * b + ⅟b * a := by
+    have : 1 = 2 + ⅟a * b + ⅟b * a := by
+      rw [← invOf_mul_self (a + b), h, add_mul, mul_add, mul_add, invOf_mul_self a,
+          invOf_mul_self b, ← add_assoc, add_assoc 1 _, add_comm 1 _, add_assoc 2 _,
+          add_comm 2 _, add_assoc, one_add_one_eq_two]
+    apply (add_left_inj 2).mp
+    conv => lhs; rw [← one_add_one_eq_two, ← add_assoc, neg_add_cancel, zero_add]
+    rw [add_comm, ← add_assoc]
+    exact this
+  let helper {x y : R} [Invertible x] [Invertible y] (h' : -1 = ⅟x * y + ⅟y * x)
+      : -(y + x) = x * ⅟y * x := by
+    apply (add_left_inj y).mp
```

Comment:
I think this is the same as 
```suggestion
    apply add_right_cancel y
```
or similar?

Reply:
`apply add_right_cancel` works!